doc/apis.adoc

```diff
@@ -44,16 +44,31 @@ It uses a `StatefulSet` with a pod spec that mounts the volume specified by `sto
 | `bootableJar` | BootableJar specifies whether the application image is using WildFly S2I Builder/Runtime images or Bootable Jar. If omitted,
 it defaults to false (application image is expected to use WildFly S2I Builder/Runtime images) | bool | false
 | `standaloneConfigMap` | spec to specify how standalone configuration can be read from a `ConfigMap` | *<<standaloneconfigmapspec>> |false
+| `resources`| Resources spec to specify the request or limits of the Stateful Set. If ommited, the namespace defaults are used. | *<<resources>> | false
```

Comment:
> or completely replace resources -> *resources by resources -> [corev1.ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#resourcerequirements-v1-core)

Yes that was my understanding what could be done to be consistent with rest.

> We could have resources -> *resources and then two fields like limits and requests

TBH that sounds to me it will be less consistent with rest.